help/en/html/doc/Technical/ContinuousIntegration.shtml

```diff
@@ -45,73 +47,29 @@
       </ul>
 
       <h2 id="mandatory">Mandatory Checks</h2>
-      We use three CI engines to test every proposed change entered into our GitHub code repository
+      We use multiple
+      <a href="https://docs.github.com/en/actions">GitHub actions</a>
+      to test every proposed change entered into our GitHub code repository
       before it gets merged.
+      <a href="images/ActionSettings.png"><img src="images/ActionSettings.png" class="floatRight" width="331" height="167" alt="github actions"></a>
```

Comment:
Maybe add a `<br>` tag before this image to get it on the next line? On my screen, the image is on the right of the sentense.
https://htmlpreview.github.io/?https://github.com/JMRI/JMRI/blob/0de225449a23f8ef1b45e44542f4650f7ea8eb89/help/en/html/doc/Technical/ContinuousIntegration.shtml

Reply:
Done. Which browser do you use?  (I should probably check with that one too...)